### ISSUE TITLE
Update UTM card on stats

### ISSUE THREAD

**Issue by fditrapani** [Automattic/wp-calypso#100504]
## Problem

The UTM card is missing an info tooltip, like all the other cards, which describes what it's for. In addition to that, when there's no information, we show a View more link that takes you to a page with no data on it.

<img width="1727" alt="Image" src="https://github.com/user-attachments/assets/10f0d6bb-d7b5-45ba-937c-5e1a34646900" />

<img width="1839" alt="Image" src="https://github.com/user-attachments/assets/ad2b4613-6641-4e62-a25b-1a6a7b8811cf" />

Let's add a tooltip and hide the view more link if there's no data to show on the subsequent page.

## Where
1. Visit the Stats screen on a site with a Premium plan or higher.
2. Scroll down to the UTM section.

**Comment by dognose24** [comment]
> The UTM card is missing an info tooltip, like all the other cards

This issue is not related to the specific UTM card, it's about the difference between empty state and cards with data.

<img width="1727" alt="Image" src="https://github.com/user-attachments/assets/5392855f-07d2-480b-bd7b-df0e8fd4b195" />

**Comment by dognose24** [comment]
We had the support link in the empty state inline text, which should have been aligned with the info icon link. If we add the info icon steadily to the title, there might be redundant information. cc @ederrengifo 

<img width="679" alt="Image" src="https://github.com/user-attachments/assets/f261298e-f5d3-45ff-8832-90b8d68dc28a" />

**Comment by kangzj** [comment]
> I don't see why we need a tooltip when the empty state already provides the support information more upfront, it seems repetitive.

I'll close this one per comment above and `View more` issue is already tracked [here](https://github.com/Automattic/wp-calypso/issues/100505).
